apps/condo/domains/common/utils/date.js

```diff
@@ -10,7 +10,7 @@ const DATE_FORMAT_Z = 'YYYY-MM-DDTHH:mm:ss.SSSZZ'
  * @param shouldFormat
  * @returns {*}
  */
-const getMonthStart = (dateRaw, shouldFormat = false) => {
+const getMonthStart = (dateRaw = undefined, shouldFormat = false) => {
```

Comment:
What's the reason of passing undefined to `dateRaw` here?

Reply:
This func is often called without any args and linter shows warnings. Undefined is normal value for it